### PR TITLE
[JS v7] **do not merge** ref(javascript): Replace deprecated severity enums with string literals

### PR DESCRIPTION
**Pull request by lforst**
In https://github.com/getsentry/sentry-javascript/pull/4926, we deprecated the `Severity`. This PR replaces all occurrences of that enum with their now recommended string value counterpart.

Ref: https://getsentry.atlassian.net/browse/WEB-849

**DO NOT MERGE: Only merge this branch after the Sentry JavaScript SDK version 7 has been published.**

### CODE REVIEW

File: src/platforms/javascript/common/troubleshooting/index.mdx [hunk 1]
```diff
@@ -389,12 +389,12 @@ document.body.addEventListener(
     if (event.target.tagName === "IMG") {
       Sentry.captureMessage(
         `Failed to load image: ${event.target.src}`,
-        Sentry.Severity.Warning
+        "warning" // message severiity level
```
Comment (Lms24) [inline src/platforms/javascript/common/troubleshooting/index.mdx:392]:
```suggestion
        "warning" // message severity level
```

File: src/platforms/javascript/common/troubleshooting/index.mdx [hunk 2]
```diff
@@ -389,12 +389,12 @@ document.body.addEventListener(
...
       );
     } else if (event.target.tagName === "LINK") {
       Sentry.captureMessage(
         `Failed to load css: ${event.target.href}`,
-        Sentry.Severity.Warning
+        "warning" // message severiity level
```
Comment (Lms24) [inline src/platforms/javascript/common/troubleshooting/index.mdx:397]:
```suggestion
        "warning" // message severity level
```

**Review by Lms24 — COMMENTED** [review]
Looking good. I wonder if we should include a list of the possible `SeverityLevel`s? Or is this self explanatory when using it - WDYT?

**Comment by lforst** [comment]
> Looking good. I wonder if we should include a list of the possible `SeverityLevel`s? Or is this self explanatory when using it - WDYT?

I was a bit on the fence about this one because other sdks don't seem to document the levels either but I think we should because otherwise it's documented nowhere except in typescript...

It was obvious with the enums but now it's not. Updated in: 2cdeaafe35c8d1b20769e2d15d9b08d73b6eaf69

**Review by Lms24 — APPROVED** [review]

**Comment by Lms24** [comment]
> because otherwise it's documented nowhere except in typescript...

Agreed. For people using just JS that might be a problem.

**Review by lobsterkatie — APPROVED** [review]
LGTM!

**Pull request merged**